src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,45 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_SIZE_STORAGE = 10;
+    private final K[] keys = (K[]) new Object[MAX_SIZE_STORAGE];
+    private final V[] values = (V[]) new Object[MAX_SIZE_STORAGE];
```

Comment:
You should use constructor to initialize this variable
```suggestion
    private final K[] keys;
    private final V[] values;
```